Gem/Code/Source/SampleComponentManager.cpp

```diff
@@ -1610,12 +1617,6 @@ namespace AtomSampleViewer
             m_xrPipelines.push_back(renderPipelineLeft);
             m_xrPipelines.push_back(renderPipelineRight);
 
-            // Disable default pipeline
-            if (!r_EnableDefaultRenderPipelineOnXR)
```

Comment:
We probably still want this code no? In case we want to disable the pipeline at runtime to get more perf. You probably would just need to wrap the code around a m_renderPipeline null check

Reply:
Done, see previous comment.